package.json

```diff
@@ -24,7 +24,7 @@
     "lint": "eslint . --cache --max-warnings 0",
     "lint:fix": "eslint . --cache --max-warnings 0 --fix",
     "typecheck": "tsc",
-    "test": "jest --detectOpenHandles",
+    "test": "jest --detectOpenHandles --runInBand",
```

Comment:
Minor, but this can make the test run slower, as they run serially not in parallel

Reply:
Added as it was recommended when running multiple tests against the pact mock server.  Can remove until/if we start getting spurious test results if you like?

Reply:
ok thanks 👍 
yep if there's a problem we can add this